Makefile

```
@@ -38,6 +38,7 @@ endif
 # Image URL to use all building/pushing image targets
 VELA_CORE_IMAGE      ?= vela-core:latest
 VELA_CORE_TEST_IMAGE ?= vela-core-test:$(GIT_COMMIT)
+VELA_RUNTIME_ROLLOUT_IMAGE       ?= vela-runtime-rollout:latest
```

```suggestion
VELA_RUNTIME_ROLLOUT_IMAGE       ?= vela-rollout:latest
```

就叫 vela-rollout， 不要强调runtime

---

其他变量里面也统一改一下